elm-app/src/elm/Contact/View.elm

```diff
@@ -65,122 +77,161 @@ viewContacts baseUrl language showAsBlock { contacts, filterString } =
         if DictList.isEmpty filteredContacts then
             div [] [ text <| translate language ContactsNotFound ]
         else
-            ul
-                [ class "list list-primary list-borders" ]
-                (filteredContacts
-                    |> DictList.map
-                        (\contactId contact ->
-                            if showAsBlock then
-                                viewContactAsBlock baseUrl language ( contactId, contact )
-                            else
-                                viewContact baseUrl language ( contactId, contact )
+            viewMaybe
+                (always <|
+                    ul [ class "list list-primary list-borders" ]
+                        (filteredContacts
+                            |> DictList.map
+                                (\contactId contact ->
+                                    viewContactAsBlock
+                                        baseUrl
+                                        language
+                                        ( contactId, contact )
+                                )
+                            |> DictList.values
+                        )
+                )
+                (always <|
+                    div [ class "row" ]
+                        (filteredContacts
```

Comment:
I know but I couldn't find a way to unify them, the HTML is different and each one is calling a different view function:
- 1st: `viewContactAsBlock`
- 2nd: `viewContact`